INSTALL.md

```diff
@@ -1,17 +1,17 @@
-# oni-ml
+# spot-ml
 
-Machine learning routines for OpenNetworkInsight, version 1.1
+Machine learning routines for Apache Spot (incubating).
 
 ## Prerequisites, Installation and Configuration
 
-Install and configure oni-ml as a part of the Open-Network-Insight project, per the instruction at
-[the Open-Network-Insight wiki](https://github.com/Open-Network-Insight/open-network-insight/wiki).
+Install and configure spot-ml as a part of the Spot project, per the instruction at
+[the Spot wiki].
```

Comment:
So, for now you are just keeping links placeholders rather than let them point to the current repo structure?